velox/exec/tests/ContainerRowSerdeTest.cpp

```diff
@@ -62,6 +95,31 @@ class ContainerRowSerdeTest : public testing::Test,
     allocator_.clear();
   }
 
+  void testCompareWithNulls(
+      const DecodedVector& decodedVector,
+      const std::vector<HashStringAllocator::Position>& positions,
+      const std::vector<std::shared_ptr<ByteStream>>& byteStreams,
+      const std::vector<int32_t>& expected,
+      std::optional<int32_t> expectedOptionVal,
+      CompareFlags compareFlags) {
+    for (auto i = 0; i < expected.size(); ++i) {
+      ASSERT_EQ(
+          expected.at(i),
+          ContainerRowSerde::compareWithNulls(
+              *byteStreams.at(i), decodedVector, i, compareFlags)
+              .value());
```

Comment:
This assumes that compareWithNulls cannot return std::nullopt? Is this a valid assumption? Don't we want to allow testing cases where compareWithNulls returns std::nullopt?

Reply:
Fixed